doc/us/manual.html

```diff
@@ -375,6 +375,10 @@ <h2><a name="mysql_extensions"></a>MySQL Extensions</h2>
   <dt><strong><code>cur:numrows()</code></strong></dt>
   <dd>See also: <a href="#cursor_object">cursor objects</a><br/>
     Returns: the number of rows in the query result.</dd>
+
+  <dt><strong><code>cur:seek(rownum)</code></strong></dt>
+  <dd>See also: <a href="#cursor_object">cursor objects</a><br/>
+    Seek to a specific row in the query result.</dd>
```

Comment:
it would be good to document that rownum is indexed from 1 and that negative values seek to the first row.